kernel/mempool.c

```diff
@@ -0,0 +1,404 @@
+#include <kernel.h>
```

Comment:
For clarity: that tag was missing when Leandro left the comment.  I added it in an updated pull request.  Github isn't terribly clear about showing when that happens.